package.json

```diff
@@ -4,22 +4,20 @@
   "private": true,
   "license": "MPL-2.0",
   "engines": {
-    "node": ">=8.0.0",
-    "yarn": ">=1.3.2"
+    "node": ">=8.10.0",
+    "yarn": ">=1.10.0"
   },
   "scripts": {
-    "build": "yarn create:fragment-matcher && neutrino build --require dotenv/config",
-    "start": "neutrino start --require dotenv/config",
-    "test": "neutrino test",
-    "lint": "neutrino lint",
-    "precommit": "neutrino stage",
-    "create:fragment-matcher": "node src/scripts/queryServer",
-    "styleguide": "neutrino styleguide:start --require dotenv/config"
+    "build": "yarn create:fragment-matcher && dotenv webpack -- --mode production",
+    "start": "dotenv webpack-dev-server -- --mode development",
+    "test": "karma start --single-run",
+    "lint": "eslint --cache --format codeframe --ext mjs,js src test",
```

Comment:
This needs to be `--ext mjs,jsx,js`

Reply:
Wow, that was silly from my end. Sorry for that.

Reply:
I didn't see it the first time I looked either 😆